### PR TITLE
support set custom redis queue name

### PR DESCRIPTION
**Pull request by leason00**
support set custom redis queue name

### CODE REVIEW

File: redis_broker.go [hunk 1]
```diff
@@ -52,6 +52,11 @@ func (cb *RedisCeleryBroker) SendCeleryMessage(message *CeleryMessage) error {
 	return nil
 }
 
+//Set custom queue name
```
Comment (sickyoon) [inline redis_broker.go:55]: this will break lint

Comment (leason00) [reply redis_broker.go:55]: Thanks for your review.
Do you mean like this？
`// RedisCeleryBroker is celery broker for redis
type RedisCeleryBroker struct {
	*redis.Pool
	QueueName string
}

// NewRedisBroker creates new RedisCeleryBroker with given redis connection pool
func NewRedisBroker(conn *redis.Pool, queueName string) *RedisCeleryBroker {
	return &RedisCeleryBroker{
		Pool:      conn,
		QueueName: queueName,
	}
}`

Comment (leason00) [reply redis_broker.go:55]: > this will break lint

Comment (sickyoon) [reply redis_broker.go:55]: yes

**Comment by sickyoon** [comment]
Sorry but I have to close this PR request. I think this should be applied to all brokers/backends.
I already merged the changes into the master: https://github.com/gocelery/gocelery/pull/144

**Pull request closed without merging**